### ISSUE TITLE
bar() prints current position on Windows

### ISSUE THREAD

**Issue by mosesontheweb** [rsalmei/alive-progress#50]
Hi,

I just installed your lib, it's great. Unfortunately I have an issue, which I can't explain.

I used your same code:
```python
with alive_bar(1000) as bar:
     for i in range(1000):
             if i % 300 == 0:
                     print('found!')
             time.sleep(0.01)
             bar()
```
Result:
```
on 0: found!
on 1: 1
on 2: 2
on 3: 3
on 4: 4
on 5: 5
on 6: 6
on 7: 7
on 8: 8
on 9: 9
on 10: 10
on 11: 11
on 12: 12
on 13: 13
on 14: 14
on 15: 15
on 16: 16
...
```
So, even though I do not have any kind of print command, there's output.

Any ideas why?

Regards, Thomas

**Comment by rsalmei** [comment]
Hello @mosesontheweb, thank you, appreciated!

Unfortunately, I can't imagine how this would be possible. Note the code doesn't even include a `print(i)` to cause that output.
Are you really sure this was the code you ran? Which version are you using? Which Python?
I've simulated it now, latest version in a Python 3.8:
![asd](https://user-images.githubusercontent.com/6652853/88293518-7caa7b00-ccd1-11ea-90c7-2353529aa459.gif)

Regards, Rogério.

**Comment by mosesontheweb** [comment]
Hi,

I'm running Python 3.6.9. Here's a cast I recorded for you: [alive_progress](https://asciinema.org/a/wsY53hn6fPiu7NBmaN28sqflS)

Regards, Thomas

**Comment by rsalmei** [comment]
Hey, that's bizarre! I don't have the slightest idea how that could happen, I'd say it's impossible without manually fiddling the source files...

Could you please try in a new virtualenv?
I like pyenv-virtualenv very very much, it just takes:
```bash
$ pyenv virtualenv 3.6.9 alive-test
$ pyenv shell alive-test
```
And this shell has that venv activated.
Then you could:
```bash
$ pip install alive-progress
$ python
```

Could you please try that?

**Comment by rsalmei** [comment]
Well, please reopen if you have more information.

**Comment by rsalmei** [comment]
Hey @mosesontheweb, how are you?

Why didn't you tell me this was occurring on Windows? It would be super relevant...

Anyway, I just installed Python on a Windows machine, and experienced this exact issue!
All I can say for now is that it is the `bar()` handle, but don't have a clue why.
I'm going to open this again, let's see if anyone with experience developing on Windows could provide some help.
![image](https://user-images.githubusercontent.com/6652853/94016880-17ccd900-fd85-11ea-9501-9699eb0387d3.png)

**Comment by rsalmei** [comment]
Humm, inside WSL (Windows Subsystem for Linux) it **does** work.
![image](https://user-images.githubusercontent.com/6652853/94039738-7783ae00-fd9e-11ea-8b80-fa89336012ed.png)

**Comment by rsalmei** [comment]
Awesome, I found it!!!!
Regardless of how bizarre it may seem, it's the return value of `bar()` that is being printed on screen!
I don't know why this Python is doing that, but just commenting that return makes all work nicely! 😄 
![image](https://user-images.githubusercontent.com/6652853/94045976-73f42500-fda6-11ea-9126-faf06d968785.png)

**Comment by rsalmei** [comment]
Ok, I've been using `ipython` and `bpython` for so long in my live that I've forgot about this quirk.
It's because of `Python`, the default REPL, that this happens. It has nothing to do with Windows.
Even on macOS I can simulate, it is because `Python` does not know how to ignore a return value. `ipython` on the other hand, interprets the statement differently, and captures only its final output:
<img width="1217" alt="image" src="https://user-images.githubusercontent.com/6652853/94047108-02b57180-fda8-11ea-99f1-12ab66ea4e8c.png">

That way, what I am going to do: I'll remove the return value by default, and create a new method called `bar.current()` that will return the current position if needed.
It will be available in the next major 2.0! See #51

**Comment by mosesontheweb** [comment]
Hi,
sorry for the late reply. I thought that I mentioned it somewhere, but it seems that I didn't. My bad. But glad to see that you were finally able to reproduce the issue and add a fix in the next major.

Regards, Thomas
